### DISCUSSION TITLE
Installation Issue Pimcore 10.x

### DISCUSSION THREAD

**Discussion by AndyHeinz** [pimcore/pimcore#9698] [Q&A / Support]
Hi,
I wanted to install pimcore on my fresh Ubuntu 20.04 VM and followed the instructions at:
https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Installation.html

The installation went through just fine.

I used the nginx conf from: 
https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/Nginx_Configuration.html

Unfortunatelly the result is a blank /admin page.

Thanks!

**Comment by HeicoSindel** [comment]
I had the same problem (only use apache instead of nginx). I solved this issue by setting the right permissions - as discribed in the documentation AND change the user (and group) within the apache-config so the user who is executing the php scripts in the same group as the one who starts the web-server. 

now i have another issue by installing the data-hub bundle - everything works fine - the bundle is shown as enabled but not activated in the settings-menu (logged in as admin)

as i also installed pimcore on ubuntu 20.04, we should maybe work on an specific install-doc for "pimcore on ubuntu" - the installation took several days related to some issues installing the php8-extensions (not all as documented are available), setting the right perms and installing bundles
